demos/tutorial/blast_io/read_assignment.cpp

```diff
@@ -8,7 +8,10 @@ using namespace seqan;
 int main(int argc, char **argv)
 {
     if (argc != 2)
-        return 1;
+    {
+      std::cout << "USAGE: FILE_IN\n";
```

Comment:
I know that this is just an example.. but.. all the usage/error messages should go to stderr.
Only the actual output of the program should use stdout. 

http://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout